languages/common-lisp/concepts/symbols/about.md

```diff
@@ -1 +1,29 @@
-TODO: add information on symbols concept
+## Quoting
```

Comment:
Might it be easier to understand quoting if instead of regards of keywords or symbols the explanation talks about evaluation? _i.e._ "quoting prevents evaluation".

Reply:
I think that's a good point! I must not have looked at this critically enough. I'll make that change now!